### ISSUE TITLE
LiveStream message action button is displayed without actions

### ISSUE THREAD

**Issue by bshimrit** [GetStream/stream-chat-react#813]
v5.0.3
I am using the MessageLivestream component. 
my allowed messageActions are ['edit', 'delete', 'react', 'reply']
when it is MY message the 3 dots option shows 'edit' + 'delete'
when it is another user it has no possible actions but the 3 dots still appear without any possible actions.
![image](https://user-images.githubusercontent.com/25264676/112815018-8ec29180-9088-11eb-97fa-fcf2796836e4.png)

**Comment by yetieaterxb1** [comment]
Hi @bshimrit,

You can only `edit` and `delete` your own messages. Another users messages will only have the option to `react` or `reply` so there are no other actions that will show up in the actions box. If you added `flag`, `mute`, or `pin` you would see actions available on another users messages.

**Comment by bshimrit** [comment]
Of course. Sorry I was unclear about the issue, that is the expected behaviou (no possible actions), but the 3 dots should not be displayed when there are no possible actions. 
When you send an empty array to messageActions, the 3 dots are not displayed. the behaviour should be the same when you have no possible actions.

**Comment by yetieaterxb1** [comment]
Hi @bshimrit,

Thanks for your input on this. I have submitted a PR that should solve this issue. 

https://github.com/GetStream/stream-chat-react/pull/816/files
